lib/trino-filesystem/src/main/java/io/trino/filesystem/UnrecoverableTrinoFileSystemException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.filesystem;
+
+import java.io.IOException;
+
+import static java.lang.String.format;
+
+/**
+ * This exception is for stopping retries for FileSystem calls that shouldn't be retried.
+ * For example, "Caused by: com.amazonaws.services.s3.model.AmazonS3Exception: Forbidden (Service: Amazon S3; Status Code: 403 ..."
```

Comment:
Remove this, since it's specific to S3 and thus not appropriate for the file system package.

Reply:
removed second line about S3